Formula/v8.rb

```diff
@@ -15,98 +17,217 @@ class V8 < Formula
+      :revision => "a325ad2db5deb623eab740527e559b81c0f39d65"
+  end
+
+  resource "external-github.com-tc39-test262" do
+    url "https://chromium.googlesource.com/external/github.com/tc39/test262.git",
+      :revision => "0192e0d70e2295fb590f14865da42f0f9dfa64bd"
   end
 
-  resource "gmock" do
+  resource "external-github.com-test262--utils-test262--harness--py" do
+    url "https://chromium.googlesource.com/external/github.com/test262-utils/test262-harness-py.git",
+      :revision => "0f2acdd882c84cff43b9d60df7574a1901e2cdcd"
+  end
+
+  resource "external-github.com-webassembly-spec" do
+    url "https://chromium.googlesource.com/external/github.com/WebAssembly/spec.git",
+      :revision => "586d34770c6445bfb93c9bae8ac50baade7ee168"
+  end
+
+  resource "external-googlemock" do
     url "https://chromium.googlesource.com/external/googlemock.git",
-        :revision => "0421b6f358139f02e102c9c332ce19a33faf75be"
+      :revision => "0421b6f358139f02e102c9c332ce19a33faf75be"
   end
 
-  resource "clang" do
-    url "https://chromium.googlesource.com/chromium/src/tools/clang.git",
-        :revision => "faee82e064e04e5cbf60cc7327e7a81d2a4557ad"
+  resource "external-gyp" do
+    url "https://chromium.googlesource.com/external/gyp.git",
+      :revision => "d61a9397e668fa9843c4aa7da9e79460fe590bfb"
   end
 
+  resource "external-llvm.org-libunwind" do
+    url "https://chromium.googlesource.com/external/llvm.org/libunwind.git",
+      :revision => "fc0a910c25d5415dd72ba9451b7cba380e3cc1e7"
+  end
+
+  resource "infra-luci-client--py" do
+    url "https://chromium.googlesource.com/infra/luci/client-py.git",
+      :revision => "88229872dd17e71658fe96763feaa77915d8cbd6"
+  end
+
+  resource "llvm--project-cfe-tools-clang--format" do
+    url "https://chromium.googlesource.com/chromium/llvm-project/cfe/tools/clang-format.git",
+      :revision => "0653eee0c81ea04715c635dd0885e8096ff6ba6d"
+  end
+
+  resource "llvm--project-libcxx" do
+    url "https://chromium.googlesource.com/chromium/llvm-project/libcxx.git",
+      :revision => "f56f1bba1ade4a408d403ff050d50e837bae47df"
+  end
+
+  resource "llvm--project-libcxxabi" do
+    url "https://chromium.googlesource.com/chromium/llvm-project/libcxxabi.git",
+      :revision => "05ba3281482304ae8de31123a594972a495da06d"
+  end
+
+  resource "v8-deps-third_party-benchmarks" do
+    url "https://chromium.googlesource.com/v8/deps/third_party/benchmarks.git",
+      :revision => "05d7188267b4560491ff9155c5ee13e207ecd65f"
+  end
+
+  resource "v8-deps-third_party-mozilla--tests" do
+    url "https://chromium.googlesource.com/v8/deps/third_party/mozilla-tests.git",
+      :revision => "f6c578a10ea707b1a8ab0b88943fe5115ce2b9be"
+  end
+
+  # not building on Yosemite
+  # https://bugs.chromium.org/p/chromium/issues/detail?id=620127
+  depends_on :macos => :el_capitan
+
+  # depot_tools/GN require Python 2.7+
+  depends_on "python@2" => :build
+
+  depends_on "readline" => :optional
```

Comment:
The formula has 1,073 installs per month, exactly zero of which use the `--with-readline` option. Let's remove it (so no users ever have to build this from source!).

Reply:
fixed